CHANGELOG.md

```diff
@@ -58,10 +88,11 @@ should change the heading of the (upcoming) version to include a major version b
 ## @rjsf/core
 
 - Updated the `MultiSchemaField` to use the new `getDiscriminatorFieldFromSchema()` API
-- Added new `experimental_defaultFormStateBehavior` prop to `Form` 
+- Added new `experimental_defaultFormStateBehavior` prop to `Form`
   - to specify alternate behavior when dealing with the rendering of array fields where `minItems` is set but field is not `required` (fixes [#3363](https://github.com/rjsf-team/react-jsonschema-form/issues/3363)) ([#3602](https://github.com/rjsf-team/react-jsonschema-form/issues/3602))
   - to handle setting object defaults based on the value of `emptyObjectFields` supporting required fields only and skipping defaults entirely, fixing [#2980](https://github.com/rjsf-team/react-jsonschema-form/issues/2980)
 - Fixed regression [#3650](https://github.com/rjsf-team/react-jsonschema-form/issues/3650) in `FileWidget` to again support adding multiple files to arrays
+- Updated SchemaField to be able to render markdown in the description field
```

Comment:
This needs to be moved up into the `5.8.0` section under `## @rjsf/core`

Reply:
Missed this one before